test-case-reporting/src/test_result_record.ts

```diff
@@ -120,89 +123,109 @@ export class TestResultRecord {
     await this.db('test_runs').insert(testRuns);
   }
 
-  /**
-   * Gets a DB.Build from the database by its build number.
-   * If there is no such build, it returns undefined.
-   * @param buildNumber The number of the Travis build, as a string.
-   */
-  private async getDbBuildFromBuildNumber(
-    buildNumber: string
-  ): Promise<DB.Build> {
-    return this.db<DB.Build>('builds')
-      .where('build_number', buildNumber)
-      .first();
-  }
+  /* eslint @typescript-eslint/camelcase: "off" */
+  private getTestRunFromRow({
```

Comment:
Since this function doesn't use `this`, let's make it a module-level function like `getDateFromTimestamp`

Reply:
Done.